modulefiles/build_hera_gnu.lua

```diff
@@ -0,0 +1,27 @@
+help([[
+This module loads libraries for building the UFS SRW App on
+the NOAA RDHPC machine Hera using GNU 9.2.0
+]])
+
+whatis([===[Loads libraries needed for building the UFS SRW App on Hera using GNU 9.2.0 ]===])
+
+prepend_path("MODULEPATH","/contrib/sutils/modulefiles")
+load("sutils")
+
+load(pathJoin("cmake", os.getenv("cmake_ver") or "3.20.1"))
+
```

Comment:
New hpc-stack location in EPIC-managed space. As discussed in [SRW-PR-419](https://github.com/ufs-community/ufs-srweather-app/pull/419), [SRW-Issue-427](https://github.com/ufs-community/ufs-srweather-app/issues/427), [UFS-WM-Issue-1465](https://github.com/ufs-community/ufs-weather-model/issues/1465), [UFS-WM-PR-1468](https://github.com/ufs-community/ufs-weather-model/pull/1468).
This stack in this location on Hera has been independently tested with the UFS-WM by @jkbk2004, and not changing the baseline. The following lines 13-16 would be changed to the following:

```
gnu_ver=os.getenv("gnu_ver") or "9.2.0"
load(pathJoin("gnu", gnu_ver))

prepend_path("MODULEPATH", "/scratch1/NCEPDEV/nems/role.epic/hpc-stack/libs/gnu-9.2/modulefiles/stack")

load(pathJoin("hpc", os.getenv("hpc_ver") or "1.2.0"))
load(pathJoin("hpc-gnu", os.getenv("hpc-gnu_ver") or "9.2"))
```